logicrunner/handle_calls.go

```diff
@@ -177,11 +200,10 @@ func (h *HandleCall) handleActual(
 		}
 
 		h.sendToNextExecutor(ctx, *objRef, *requestRef, request, pendingState)
+		return repl, nil
```

Comment:
there is not point in this return

Reply:
there is, it's a ground for my refactoring :)))) and side effect of rebasing 4 times.

Reply:
deleted return